src/domains/transaction/components/FeeField/FeeField.tsx

```diff
@@ -79,7 +79,7 @@ export const FeeField: React.FC<Properties> = ({ type, network, profile, ...prop
 			return true;
 		}
 
-		if (type === "delegateRegistration" && !data.username) {
+		if (type === "validatorRegistration" && !data.username) {
```

Comment:
is the username check still relevant? validators no longer have a username field

Reply:
Correct, replaced it with `validatorPublicKey`